doc/GUIDE.md

```diff
@@ -193,12 +223,15 @@ public static MethodHandle foo$handle() { ... } // 4
 
 First and foremost, there is a static wrapper method that is generated that can be used to
 call the C function (1). Besides that, there are also several accessors that return
-additional meta-data for the method: the function's address (2), the function descriptor
-(3), and the method handle returned by the FFM linker (4), which is used to implement the
-static wrapper method (1).
-
-The parameter types and return type of this method depend on the carrier types of the
-layouts that make up the function descriptor of the function, which is itself derived
+additional meta-data for the method: the function's address, represented as a
```

Comment:
Should we use a numbered list here too?

Reply:
I don't think it's possible to start a markdown list at `2`, but I will try

Reply:
Looks like this doesn't work unfortunately.

Reply:
No, wait, got it working after all.